pkg/autoscaler/autoscaler.go

```diff
@@ -80,7 +86,7 @@ func (b statsBucket) concurrency() float64 {
 	for _, podStats := range b {
 		var subtotal float64
 		for _, stat := range podStats {
-			subtotal += stat.AverageConcurrentRequests
+			subtotal += stat.AverageConcurrentRequests - stat.AverageProxiedConcurrentRequests
```

Comment:
Could you add some comments to explain why we need to discount proxied concurrency?

Reply:
Done.